docs/codeql/codeql-cli/creating-codeql-databases.rst

```diff
@@ -221,6 +221,23 @@ commands that you can specify for compiled languages.
 
      codeql database create java-database --language=java --command='ant -f build.xml'
 
+- Project built using Bazel::
+
+     # Navigate to the Bazel workspace.
+
+     # Stop all running Bazel server processes.
+     bazel shutdown
```

Comment:
I think we need this _after_ the build, too, because the existing bazel server process will have the tracer attached to and configured - that'll dump trap in your database when you compile something else

Reply:
However, before the build it is not necessary:

> When executed with --expunge, the clean command simply removes the entire output base tree which, in addition to the build output, contains all temp files created by Bazel. It also stops the Bazel server after the clean, equivalent to the shutdown command.

Source: https://docs.bazel.build/versions/4.2.1/user-manual.html#the-clean-command